packages/connect-voting-disputable/src/models/Vote.ts

```diff
@@ -50,12 +62,18 @@ export default class Vote {
     this.quietEndingExtendedSeconds = data.quietEndingExtendedSeconds
     this.quietEndingSnapshotSupport = data.quietEndingSnapshotSupport
     this.script = data.script
+    this.executedAt = data.executedAt
+  }
+
+  get hasEnded(): boolean {
+    const now = new Date().getTime()
+    return now >= toMilliseconds(this.endDate)
   }
 
   get endDate(): string {
-    const originalEndDate = BigNumber.from(this.startDate).add(BigNumber.from(this.duration))
-    const endDateAfterPause = originalEndDate.add(BigNumber.from(this.pauseDuration))
-    return endDateAfterPause.add(BigNumber.from(this.quietEndingExtendedSeconds)).toString()
+    const originalEndDate = bn(this.startDate).add(bn(this.duration))
+    const endDateAfterPause = originalEndDate.add(bn(this.pauseDuration))
+    return endDateAfterPause.add(bn(this.quietEndingExtendedSeconds)).toString()
```

Comment:
Wonder if we also want to convert the values with `toMilliseconds` so the consumer should always expect milliseconds from the connectors.

Reply:
Yes, I think that's a great idea. Although we could change all the models expose in connect at once to make sure there is no hybrid behavior, what do you think?

Reply:
Good point, let's do that!